app/views/admin/poll/questions/show.html.erb

```diff
@@ -6,9 +6,9 @@
 <div class="clear"></div>
 
 <div class="row">
-  <div class="small-12 medium-9 column">
+  <div>
```

Comment:
no empty `<div>` 🙏  if you wanna 100% width use `small-12 column` and also if you wanna remove side paddings, remove `column` class 😉